### DISCUSSION TITLE
I need a video tutorial  :(

### DISCUSSION THREAD

**Discussion by q531620267** [wurstscript/WurstScript#1032] [General]
I have completed a map using the GUI. I want to learn wurst language. I use Google translation to learn web tutorial. I have realized the creation of dialog box based on Hello world script. Now I want to make a double-click unit to select heroes, but I haven't made any progress after studying for a day. Where can I see a video tutorial that can let me know how to make a hero skill, or let me understand some basic principles? I hope a video can guide me. Thank you.

**Comment by Frotty** [comment]
This doesn't really belong in the issue tracker.
This post is a little bit outdated, but we made a spell in the blog https://wurstlang.org/blog/bestofthewurst5.html
Also consider joining the [Discord](https://discord.gg/mSHZpWcadz) if you need help.
I can offer video tutorials or custom tutoring, but that would of course cost you money (I'm busy and we already invested years of our time for free in this project).

**Reply by peq** [reply]
How much for a video tutorial? 😁

**Reply by q531620267** [reply]
> This doesn't really belong in the issue tracker. This post is a little bit outdated, but we made a spell in the blog https://wurstlang.org/blog/bestofthewurst5.html Also consider joining the [Discord](https://discord.gg/mSHZpWcadz) if you need help. I can offer video tutorials or custom tutoring, but that would of course cost you money (I'm busy and we already invested years of our time for free in this project).

OK, thank you. Let me have a serious look.
